sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -255,7 +255,7 @@ public CosmosClientBuilder setConsistencyLevel(ConsistencyLevel desiredConsisten
      *
      * @return the connection policy
      */
-    public ConnectionPolicy getConnectionPolicy() {
```

Comment:
If these methods are not public on builder, and we are not providing getters on the client, How can enduser get the config of the client after creating it?

Reply:
They don't need to.. We discussed it, and end user should not need to get the config of the client they built. 
Just in case if they need it, we will make them available on the client itself, and not on the builder.

Reply:
I strongly feel it will be good to have these getters atleast on the client. I would have missed the discussion, but even .NET sdk has these getters public https://github.com/Azure/azure-cosmos-dotnet-v3/blob/f2e9d97d176fecdbe186c13575db06f7539b75e3/Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs#L161
This can be done separately out of this PR if required but should be tracked.

Reply:
Yeah, I thought so too, but I couldn't present a solid argument, as in why end user needs them, since end user is already setting these values on the builder. If they need to access it again, they can get it from the place they initially set it in the builder. 
Let me know if you have a strong use case, may be I am missing something.